stdlib/socket/src/main/ballerina/socket/Module.md

```diff
@@ -0,0 +1,63 @@
+## Module overview
+This module provides an implementation for connecting to remote socket server or act as a server for incoming socket request. The module facilitates two types of endpoints as `Client` and `Listener`.
```

Comment:
Made some minor fixes. Change to the following:

"This module provides an implementation for connecting to a remote socket server or acts as a server for incoming socket request. This module facilitates two types of endpoints called `Client` and `Listener`."